sysroot_creator/scripts/create_cc_sysroot

```diff
@@ -81,10 +83,11 @@ class DockerConfig:
     3. Setting to enable/disable caching during docker build
     """
 
-    def __init__(self, base_image, network_mode, nocache):
+    def __init__(self, base_image, network_mode, nocache, ros2_workspace):
         self.base_image = base_image
         self.network_mode = network_mode
         self.nocache = nocache
+        self.ros2_workspace = ros2_workspace
```

Comment:
If this is not meant to be exposed, it'd be better to make this protected. (`self._ros2_workspace`)